docs/src/main/asciidoc/maven-tooling.adoc

```diff
@@ -354,6 +343,8 @@ Now you should be able to execute `./mvnw quarkus-bootstrap:build-tree` on your
 ...
 ----
 
+The goal the `mode` parameter whose default value is `prod`, i.e. the production build. Alternatively, it accepts values `test` to display the test dependency tree and `dev` to display the dev mode dependency tree.
```

Comment:
Is it just me or is the beginning of this sentence a bit strange?

Reply:
No, it's not right. Thanks.

Reply:
Fixed.